components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/mediasession/GeckoMediaSessionDelegate.kt

```diff
@@ -42,7 +43,8 @@ internal class GeckoMediaSessionDelegate(
                     withTimeoutOrNull(ARTWORK_RETRIEVE_TIMEOUT) {
                         it.getBitmap(ARTWORK_IMAGE_SIZE).await()
                     }
-                } catch (e: IllegalArgumentException) {
+                } catch (e: RuntimeException) {
+                    // TODO narrow this once GeckoView moved to ImageProcessingException
```

Comment:
Keep a reference to the bug for this in the comment so we can come back to it and refer for later.

Reply:
Agreed.  I'll ask for a Bugzilla issue and add it to the comments.  Thanks,

Reply:
Feel free to file it yourself and let whoever know. I'm sure no one would mind. :)

Reply:
Created the issue https://bugzilla.mozilla.org/show_bug.cgi?id=1691072.  Updated the dcomment as well.  Thanks,

Reply:
Updated to https://bugzilla.mozilla.org/show_bug.cgi?id=1689745